docker-compose.dev.yml

```diff
@@ -7,6 +7,10 @@ services:
   db:
     ports:
       - "8890:8890"
+  mocklogin:
```

Comment:
I'm not sure if the dev environment uses the docker-compose.dev.yml file, and I think we need this functionality there as well, right?

Maybe this should be added to the base file instead, and then disabled in the docker-compose.overrides.yml on the server? Or do we also add this in the override file?

Reply:
Also, I would dasherize the service name :smile:

Reply:
for now it can stay like this. once we figured out how to make it work in the frontend, we'll see if it needs to be added in the docker compose yaml file. i did it like that because we might have a docker.compose.demo.yml for the demo environment.

Reply:
naming convention in locket is lower case:
![image](https://user-images.githubusercontent.com/3816305/108207423-02878b00-7128-11eb-9da3-a74c9be327ac.png)

Reply:
I don't think that's an actual naming convention. Seems more like an oversight to me. The name is dasherized in the service name and the front-end route. So I would dasherize it here as well, even if it's not the same in the other apps.

Reply:
the service name corresponds to the hostname. I've never dasherized any of my docker-compose service and I never seen it anywhere. if it's a convention and if there is a good reason to do that, I'll do it. otherwise I don't see the point;

Reply:
Like I said, I don't think there is an actual convention about it. But making service names consistent seems like a good thing to me. Besides consistency there is no point.

Also, I opened a [random config file](https://github.com/lblod/app-digitaal-loket/blob/213c3693dfa195307d9b6f02464f1e50f80a1a55/docker-compose.yml#L27) and while there are exceptions, it seems most multi word names are dasherized.

Anyway, I won't force you. Keep it like this if you want. I just gave you my opinion :smile:.

Reply:
Sure, it's opinated. code review maybe should be focused on the feature itself to prevent possible bugs, but it's also just my opinion  😅

Also, in the delta notifier doc, there is no dashes. see the readme => https://github.com/mu-semtech/delta-notifier

![image](https://user-images.githubusercontent.com/3816305/108211083-98251980-712c-11eb-857e-412f8d3aedc1.png)

Reply:
Yea like I said, there seem to be exceptions :smile:. The delta notifier was also listed in that yml file but the general trend seemed to be dasherizing. I agree, ideally code review should only about features and not nitpicky stuff like this. That's why I like linters and formatters and naming conventions. I don't think we should have a "don't care" mentality if there are no rules set in stone though. Dasherizing the name seemed logical to me, which is why I made this suggestion.

Anyway, we are wasting time here. Name your services however you want, I'm still going to frown every time I see someone lowercase multi word names in code :smile: